roles/st2mistral/tasks/main.yml

```diff
@@ -55,9 +42,29 @@
     - restart mistral
   tags: st2mistral
 
-- name: Setup Mistral DB tables, etc
+- name: Make sure "Initiate mistral database" doesn't run twice
```

Comment:
You're creating the file here, but I don't see where you're checking for the presence of the file in the previous task (i.e. no `when` clause)